crypto.pl

```diff
@@ -33,7 +33,12 @@
 */
 
 :- module(crypto,
-          [ evp_decrypt/6,              % +CipherText, +Algorithm, +Key, +IV, -PlainText, +Options
+          [ crypto_term_hash/3,         % +Data, -Hash, +Options
```

Comment:
Is it customary to refer to atom|string|chars|codes as 'data'?  A number is also data.  A stream and a file also contain data.

BTW the documentation of `crypto_data_hash/3` talks about a "code-list".  This is not the same as "a list of characters" or are both supported?